client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -31,34 +31,39 @@ const synchronizationOptions: CheckboxOptionItem[] = [
 	{
 		name: 'themes',
 		label: translate( 'Themes' ),
+		subTitle: translate( 'Everything in the themes directory.' ),
 		checked: false,
 		isDangerous: false,
 	},
 	{
 		name: 'plugins',
 		label: translate( 'Plugins' ),
+		subTitle: translate( 'Everything in the plugins directory.' ),
 		checked: false,
 		isDangerous: false,
 	},
 	{
 		name: 'uploads',
 		label: translate( 'Media Uploads' ),
+		subTitle: translate(
+			'Everything in the media library. You must also select ‘Site database‘ for restored media uploads to appear.'
```

Comment:
I'm not sure about the second sentence here: should we say it? Is rebuilding media using CLI or a plugin still possible, so users potentially could synchronize media without synchronizing the database?

Reply:
```suggestion
			'All files in the uploads directory. You must also select ‘Site database‘ if it is necessary for the files to appear as media uploads in WordPress.'
```